src/main/java/com/ca/apim/gateway/cagatewayconfig/util/gateway/BundleElementNames.java

```diff
@@ -0,0 +1,29 @@
+package com.ca.apim.gateway.cagatewayconfig.util.gateway;
+
+/**
+ * Utility class to hold element names used in gateway bundles.
+ */
+public class BundleElementNames {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=62&issues=AWVkDSPa4Bfu85TPS_PK&open=AWVkDSPa4Bfu85TPS_PK)

[ISSUE_KEY]: AWVkDSPa4Bfu85TPS_PK